sbin/xivo-check-master-status

```diff
@@ -9,7 +9,11 @@ if [ -z "$ip_addr" ]; then
 fi
 
 ping -c $ping_count -i $ping_interval $ip_addr
-if [ $? -eq 0 ]; then
+ping_status=$?
+if [ ! -f /var/lib/wazo/is-secondary ]; then
+    echo "HA mode changed, this stack is no longer a secondary" >&2
```

Comment:
What is the usage of this echo/stdout redirect? This script is used in a cron, so redirecting stdout to stderr will trigger an email to the admin

Reply:
Good point. The point is to be informative about the behavior. I think this is useful when using the command manually(e.g. for tests), and if we(or the administrator) decide to process the output of cron commands. I think it can be worth it to deal with command outputs in cron to help with understanding system behavior and make debugging easier, e.g. by piping the output through `logger`(rsyslog) or similar.

Reply:
Oh ok, got it.